quickwit-cli/src/cli.yaml

```diff
@@ -12,17 +12,24 @@ subcommands:
                 about: Creates an index.
                 display_order: 1
                 args:
-                    - metastore-uri:
-                        about: Location of the metastore.
-                        long: metastore-uri
-                        value_name: METASTORE URI
+                    - index-id:
+                        about: ID of the target index.
+                        long: index-id
+                        value_name: INDEX ID
                         required: true
-                        env: QUICKWIT_METASTORE_URI
                     - index-config-uri:
                         about: Location of the index config file.
                         long: index-config-uri
                         value_name: INDEX CONFIG URI
                         required: true
+                    - index-uri:
+                        about: URI of the target index.
```

Comment:
I think we need to explain this is the `storage of the index's splits`.
Index-uri might sound like a URI to access (read & search) the index.

Reply:
done.